example/src/main/java/example/Main.java

```diff
@@ -75,6 +77,22 @@ private void doStuff() {
       logger.info("hi there {}", fb -> fb.only(fb.person("person", abe)));
     }
 
+    // If you have conditions that are CPU expensive or may block on I/O network etc,
+    // then you can delegate logging to another thread (MDC / ThreadContext is copied).
+    final ExecutorService executor =
+        Executors.newSingleThreadExecutor(
+            r -> {
+              Thread t = new Thread(r);
+              t.setDaemon(true);
```

Comment:
should mention threads should be daemon?  Shut down executors on logger context stop?